torch/utils/_foreach_utils.py

```diff
@@ -5,6 +5,59 @@
 from torch import Tensor
 from torch.autograd.grad_mode import no_grad
 
+__all__ = ["SupportForeachDevices"]
+
+class SupportForeachDevices(object):
```

Comment:
Bikeshed on naming: ForeachSupportedDevices perhaps?
```suggestion
class ForeachSupportedDevices(object):
```